konrad/cloud.py

```diff
@@ -19,36 +19,57 @@ def get_p_data_array(values, units='kg m^-2', numlevels=200):
     elif type(values) is np.ndarray:
         if values.shape == (numlevels,):
             return DataArray(values, dims=('mid_levels',),
-                         attrs={'units': units})
+                             attrs={'units': units})
         else:
-            raise ValueError('Cloud parameter input array is not the right size'
-                             ' for the number of model levels.')
+            raise ValueError(
+                'Cloud parameter input array is not the right size'
+                ' for the number of model levels.')
 
     elif type(values) is int or type(values) is float:
-        return DataArray(values*np.ones(numlevels,),
+        return DataArray(values * np.ones(numlevels, ),
                          dims=('mid_levels',),
                          attrs={'units': units})
 
     raise TypeError(
-            'Cloud variable input must be a single value, numpy.ndarray or a '
-            'sympl.DataArray')
+        'Cloud variable input must be a single value, `numpy.ndarray` or a '
+        '`sympl.DataArray`')
 
 
-def get_waveband_data_array(values, units='dimensionless', numbands=14,
-                            numlevels=200, sw=True):
+def get_waveband_data_array(values, units='dimensionless', numlevels=200,
+                            sw=True):
     """Return a DataArray of values."""
     if type(values) is DataArray:
         return values
 
```

Comment:
I know this was my code in the first place (sorry), but I guess it's better to use `isinstance()` instead of `if type() is`...